src/mintpy/prep_nisar.py

```diff
@@ -61,16 +61,16 @@ def load_nisar(inps):
     print(f"Found {len(input_files)} unwrapped files")
 
     if inps.subset_lat:
-        bbox = (inps.subset_lat[0], inps.subset_lon[0], inps.subset_lat[1], inps.subset_lon[1])
+        bbox = (inps.subset_lon[0], inps.subset_lat[0], inps.subset_lon[1], inps.subset_lat[1])
     else:
-        bbox=None
+        bbox = None
 
     # extract metadata
     metadata, bounds = extract_metadata(input_files, bbox)
 
     # output filename
-    stack_file = os.path.join(inps.out_dir, "inputs/ifgramStack.h5")
-    geometry_file = os.path.join(inps.out_dir, "inputs/geometryGeo.h5")
+    stack_file = os.path.join("./inputs/ifgramStack.h5")
+    geometry_file = os.path.join("./inputs/geometryGeo.h5")
```

Comment:
The reason I removed it was a confusing path set in load_data where it pointed to '../mintpy' and it would show an error if you select another folder name. But you are right, I could keep this general format and fix it in load_data